src/assets/index.js

```diff
@@ -33,6 +33,61 @@ $('#about_btn').click(function() {
     $('#about_content').toggle('slow');
 });
 
+analysis_report_json = {
+    Disease: "Pepper Bell Healthy",
+    Symptoms: {
+        1: "Initial symptoms of infection are the formation of small, circular, water-soaked spots on leaves, stems, petioles and/or peduncles",
+        2: "Infected Pepper Bell have Circular lesions on fruit which contain tan to orange to black concentric rings in the center.",
+        3: "It can have lesions may also occur on leaves and stems and appear as irregularly shaped gray spots with dark margins",
+        4: "Seeds did not germinate; seedlings collapsing and dying; dark stems which are shriveled near the soil line",
+        5: " Water-soaked lesions on the stem and discolored roots.",
+        6: "High numbers of lesions may form on leaves causing them to turn yellow and drop from the plant.",
+    },
+    Treatment: {
+        1: "Plant only diseasefree, certified seed",
+        2: "Always plant disease-free seeds and transplants.",
+        3: "Seeds can be freed from infection by treating with hot water.",
+        4: "Use disease free planting material; remove and destroy all crop debris after harvest, or plow material deeply under soil",
+        5: "Magnesium deficiency can be prevented by applying dolomite lime to the soil, if an increase in soil pH is required, or through applications of a fertilizer containing magnesium.",
+    },
+    Recommended_Product: {
+        1: "Chlorothalonil-720-SFT : https://www.amazon.com/Chlorothalonil-Generic-Daconil-weatherstik-quali-1060/dp/B004GTOKSO",
+        2: " Tafgor-Dimethoate : https://www.amazon.in/Tata-TATA-Tafgor-Dimethoate-Insecticide/dp/B074CCXPKF",
+        3: "Ethion Insecticide : https://www.indiamart.com/proddetail/ethion-insecticide-12777127212.html",
+        4: "Vector Super : https://www.amazon.in/Vector-100ML-IMIDACLOPRID-Systemic-Insecticide/dp/B07D7YTYTB",
+    },
+};
+
+const form = document.querySelector(".upload-form");
+form.addEventListener("submit", (e) => {
+    e.preventDefault();
+    show_result_container();
+});
+
+const show_disease = document.querySelector("#disease");
+show_disease.textContent = analysis_report_json.Disease;
+
+const show_symptoms = document.querySelector("#symptoms");
+const symptoms = Object.values(analysis_report_json.Symptoms);
+symptoms.forEach((item) => {
+    const list = document.createElement("li");
+    list.textContent = item;
+    document.querySelector("#symptoms").appendChild(list);
+});
+
+const treatment = Object.values(analysis_report_json.Treatment);
+treatment.forEach((item) => {
+    const list = document.createElement("li");
+    list.textContent = item;
+    document.querySelector("#treatment").appendChild(list);
+});
+
+const product = Object.values(analysis_report_json.Recommended_Product);
+product.forEach((item) => {
+    const list = document.createElement("li");
+    list.textContent = item;
```

Comment:
create anchor tags for product, check how products are shown 

and move all this code for updating html in a function named update_result, with a parameter named analysis_report_json, 

and call update_result function from the form submit event listener, and comment above that 
//on successful response